prac_07/programming_language.py

```diff
@@ -0,0 +1,21 @@
+class ProgrammingLanguage:
+    """CP1403 programming language class"""
+
+    def __init__(self, name: str = "", typing: str = "", reflection: bool = False, year: int = "0000"):
```

Comment:
PyCharm should be warning you about extra spaces in this line. Reformat the code to see what it changes.

Reply:
Fixed:
def __init__(self, name: str="", typing: str="", reflection: bool=False, year: int="0000"):